driver.go

```diff
@@ -327,35 +348,92 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 		return nil, nil, fmt.Errorf("image name required")
 	}
 
+	img, err := d.podmanClient.InspectImage(driverConfig.Image)
+	if err != nil {
+		return nil, nil, fmt.Errorf("image %s couldn't be inspected", driverConfig.Image)
+	}
+	d.logger.Debug("Image", fmt.Sprintf("%#v", img))
+	d.logger.Debug("Image config", fmt.Sprintf("%#v", img.Config))
+
 	allArgs := []string{driverConfig.Image}
+	command := img.Config.Cmd
+	// Use the images default command, unless the user overrides
+	// it by setting the "command" parameter in the task config
 	if driverConfig.Command != "" {
-		allArgs = append(allArgs, driverConfig.Command)
+		command = []string{driverConfig.Command}
 	}
+	allArgs = append(allArgs, command...)
 	allArgs = append(allArgs, driverConfig.Args...)
+
+	entryPoint := strings.Join(img.Config.Entrypoint, " ")
```

Comment:
@weeezes  i did a few tests and i think it's not necessary to inspect the image in order to preset entryPoint/workingDir. Seems like podman does "the right thing" if one simply populates createOpts.EntryPoint = nil / createOpts.WorkDir = nil.  Can you please verify this finding on your machine? 
Important is to set it to nil, not to empty string.

Reply:
Nice find, confirmed that that indeed works! Pushed a change for it :).